src/App.tsx

```diff
@@ -32,13 +32,17 @@ function App() {
 					<Routes>
 						<Route element={<HomePage />} path="/" />
 						<Route element={<ResultLayout />} path=":network">
+							<Route element={<LatestExtrinsicsPage />} index />
 							<Route element={<ExtrinsicPage />} path="extrinsic/:id" />
 							<Route element={<SearchPage />} path="search" />
 							<Route element={<BlockPage />} path="block/:id" />
 							<Route element={<CallPage />} path="call/:id" />
 							<Route element={<AccountPage />} path="account/:address" />
 							<Route element={<EventPage />} path="event/:id" />
-							<Route element={<LatestExtrinsicsPage />} path="latest-extrinsics" />
+							<Route element={<Navigate to={
+								window.location.pathname
+									.substring(0, window.location.pathname.substring(1).indexOf("/") + 1)
+							} replace />} path="latest-extrinsics" />
```

Comment:
Tak jsem zjistil ze v tomhle nasem pripade nejlip poslouzi relativni cesta tedy muzes pouzit `<Navigate to=".." />`